digital_image_processing/test_digital_image_processing.py

```diff
@@ -91,3 +94,21 @@ def test_nearest_neighbour(
     nn = rs.NearestNeighbour(imread(file_path, 1), 400, 200)
     nn.process()
     assert nn.output.any()
+
+
+def test_local_binary_pattern():
+    file_path: str = "digital_image_processing/image_data/lena.jpg"
+
+    # Reading the image and converting it to grayscale.
+    image = imread(file_path, 0)
+
+    # Create a numpy array as the same height and width of read image
+    lbp_image = np.zeros((image.shape[0], image.shape[1]))
+
+    # Iterating through the image and calculating the local binary pattern value
+    # for each pixel.
+    for i in range(0, image.shape[0]):
+        for j in range(0, image.shape[1]):
+            lbp_image[i][j] = lbp.local_binary_value(image, i, j)
+
+    assert lbp_image.any()
```

Comment:
How does this assertion prove that the algorithm is doing what it is supposed to do?
Maybe it would be more rigorous to test each function individually.